app/scripts/views/complete_sign_up.js

```diff
@@ -65,6 +69,7 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       return self.fxaClient.verifyCode(uid, code)
           .then(function () {
             self.logScreenEvent('verification.success');
+            self._verificationExperiment.performAction('verification');
```

Comment:
Will change name in the refactor, might not need this if we are moving logic out of the experiments